src/Input/Silk.NET.Input.Sdl/SdlKeyboard.cs

```diff
@@ -60,13 +60,14 @@ public unsafe void DoEvent(Event @event)
                 }
                 case EventType.Textinput:
                 {
-                    var chars = stackalloc char[32];
-                    Encoding.UTF8.GetChars(&@event.Text.Text[0], 32, chars, 32);
-                    int i = 0;
-                    // run the KeyChar event until we get a null terminator
-                    while (chars[i] != default)
+                    if (KeyChar != null)
```

Comment:
nit: prefer identity comparisons

```suggestion
                    if (KeyChar is not null)
```